Earthfile

```diff
@@ -238,7 +238,7 @@ earthly:
     ARG VERSION="dev-$EARTHLY_TARGET_TAG_DOCKER"
     ARG EARTHLY_GIT_HASH
     ARG DEFAULT_BUILDKITD_IMAGE=docker.io/earthly/buildkitd:$VERSION # The image needs to be fully qualified for alternative frontend support.
-    ARG BUILD_TAGS=dfrunmount dfrunsecurity dfsecrets dfssh dfrunnetwork dfheredoc
+    ARG BUILD_TAGS=dfrunmount dfrunsecurity dfsecrets dfssh dfrunnetwork dfheredoc forceposix
```

Comment:
From @GMaxera:
> The https://github.com/jessevdk/go-flags package automatically uses the / for options when detect is building on Windows. This is, of course, undesired in the case of Earthfile. The forceposix makes go-flags always use the dashes for the options.

This seems to check out with the docs here: https://pkg.go.dev/github.com/jessevdk/go-flags#hdr-Supported_features

@alexcb what do you think?

Reply:
To be clear, this fixes a third issue happening on Windows machines. When in an Earthfile you write a command like for example:
```
SAVE ARTIFACT --force ./some/path AS ...
```
without the "forceposix", the parser will expect `/force` to specify flags and it will mistake `/some/path` as a flag.
With "forceposix" the file is correctly interpreted in the same (exact) way both on Windows and non-Windows environments.